### PR TITLE
[MB-14533] capture updates to customer information

### PR DESCRIPTION
**Pull request by ruizajtruss**
## Summary

Two bugs are being handled in this PR. The first is surrounding customer name updates done by a service counselor not being captured. That was due to a missing template which has now been added. The second was a similar bug; but for the backup contact. Which was resolved by adding a wildcard event name to the pre-existing template `updateServiceMemberBackupContact`. I went with that approach since the only difference in the code would be the event name, so writing an entirely new template felt unnecessary.

## Setup to Run Your Code

<details>
<summary>💻 You will need to use two separate terminals to test this locally.</summary>

##### Terminal 1

Start the UI locally.

```sh
make client_run
```

##### Terminal 2

Start the Go server locally.

```sh
make server_run
```

</details>

### Additional steps

1. Login as a services counselor
2. Find a move (I used `8B63JG`)
3. In the move details page, scroll down and click "Edit customer info"
4. Update a field for the customer name
5. update a field for the backup contact
6. Click "Save"
7. Should display in the move history tab

## Verification Steps for Author

These are to be checked by the author.

- [x] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Frontend

- [ ] User facing changes have been reviewed by design.
- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/main/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
- [ ] There are no new console errors in the browser devtools
- [ ] There are no new console errors in the test output
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

## Screenshots
Updating Customer Name Before:
image

Updating Customer Name After 🎉 :
image

Updating Backup Contact Before:
image

Updating Backup Contact After 🎉 :
image

### CODE REVIEW

File: src/constants/MoveHistory/EventTemplates/UpdateCustomer/updateCustomer.test.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import updateCustomer from 'constants/MoveHistory/EventTemplates/UpdateCustomer/updateCustomer';
+
+describe('When a service counselor updates a customer name', () => {
+  const historyRecord = {
+    action: 'UPDATE',
+    eventName: 'updateCustomer',
+    tableName: 'service_members',
+    eventNameDisplay: 'Updated profile',
+    changedValues: {
+      first_name: 'Chuck',
```
Comment (mmayo-truss) [inline src/constants/MoveHistory/EventTemplates/UpdateCustomer/updateCustomer.test.jsx:13]: Praise: Solid choice of test case reference.

File: src/constants/MoveHistory/EventTemplates/UpdateServiceMemberBackupContact/updateServiceMemberBackupContact.test.jsx [hunk 2]
```diff
@@ -14,8 +12,8 @@ const BACKUP_CONTACT = {
 describe('When a service members updates their profile', () => {
   const template = {
     action: 'UPDATE',
-    eventName: o.updateServiceMemberBackupContact,
-    tableName: t.backup_contacts,
+    eventName: 'updateServiceMemberBackupContact',
```
Comment (mmayo-truss) [inline src/constants/MoveHistory/EventTemplates/UpdateServiceMemberBackupContact/updateServiceMemberBackupContact.test.jsx:15]: Note: looks good, though I think you might have conflicts with @afoon 's Pr https://github.com/transcom/mymove/pull/9635

**Review by mmayo-truss — APPROVED** [review]
Looks good to me! Good work

File: src/constants/MoveHistory/EventTemplates/UpdateServiceMemberBackupContact/updateServiceMemberBackupContact.test.jsx (hunk 2 above)
Comment (ruizajtruss) [reply src/constants/MoveHistory/EventTemplates/UpdateServiceMemberBackupContact/updateServiceMemberBackupContact.test.jsx:15]: Double-checked and it seems like it should be okay 🎉

**Pull request merged**